java/arcs/core/storage/driver/Database.kt

```diff
@@ -100,6 +100,18 @@ object DatabaseDriverProvider : DriverProvider {
         manager.removeEntitiesCreatedBetween(startTimeMillis, endTimeMillis)
     }
 
+    override suspend fun getEntitiesCount(inMemory: Boolean): Long {
+        return manager.getEntitiesCount(!inMemory)
```

Comment:
this is a bit confusing. I suggest either using persistent everywhere, or using inMemory everywhere

Reply:
From counting perspective, the important dimension is whether the storage is in-memory or on-disk. Drivers needs to translate these into their own language (persistent here) and here is where the translation happens.